addons/mail/static/src/model/model_field.js

```diff
@@ -129,6 +130,10 @@ class ModelField {
          * relation is removed, the related record is automatically deleted.
          */
         this.isCausal = isCausal;
+        /**
+         * Determine whether the field is read only or not.
```

Comment:
Please explain what we do with read only in details here (spec in pad with every field type).